recipe/meta.yaml

```diff
@@ -1,25 +1,23 @@
-{% set name = "TPOT" %}
 {% set version = "0.9.5" %}
 {% set sha256 = "84998cba68a8691a8a563df619558cc8943e1c9f1807ebc4de555558ff465b68" %}
 
 package:
-  name: {{ name|lower }}
+  name: tpot
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/T/TPOT/TPOT-{{ version }}.tar.gz
   sha256: {{ sha256 }}
 
 build:
-  number: 0
+  number: 1
   noarch: python
```

Comment:
Is the `- pywin32  # [win]` requirement necessary? If yes, please remove `noarch: python`. If not, please remove `- pywin32  # [win]`

Reply:
It seems `pywin32` is a legit dependency:
https://github.com/EpistasisLab/tpot/blob/507b45db01e8f88651f4ce8e03b607a5b50146f5/tpot/base.py#L75-L94

Reply:
Thanks for checking. I'll change the recipe accordingly.